packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1261,9 +1268,11 @@ export class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
     return iteration
       .next(() => PersistencePromise.waitFor(promises))
       .next(() =>
-        this.db.getRemoteDocumentCache().updateSize(txn, -bytesRemoved)
+        this.db
+          .getRemoteDocumentCache()
+          .updateMetadata(txn, changedKeys, -bytesRemoved)
```

Comment:
So it seems like before we weren't properly propagating changedKeys to the documentChangesStore, but now we are?  I assume that was essentially a bug?